### PR TITLE
new template for web app and github

### PR DESCRIPTION
**Pull request by tfitzmac**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@tfitzmac__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (tomfitz). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: app-service-web-arm-from-github-provision/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      "resources":[
+        {
+          "apiVersion":"2015-04-01",
+          "name":"web",
+          "type":"sourcecontrols",
+          "dependsOn":[
+            "[resourceId('Microsoft.Web/Sites', parameters('siteName'))]"
+          ],
+          "properties":{
+            "RepoUrl":"https://github.com/davidebbo-test/Mvc52Application.git",
```
Comment (singhkay) [inline app-service-web-arm-from-github-provision/azuredeploy.json]: This should probably become a parameter

File: app-service-web-arm-from-github-provision/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            "branch": "master",
```
Comment (singhkay) [inline app-service-web-arm-from-github-provision/azuredeploy.json]: This one as well a parameter

File: app-service-web-arm-from-github-provision/readme.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Deploy a web app linked to a GitHub repository
```
Comment (singhkay) [inline app-service-web-arm-from-github-provision/readme.md]: Suggestion for name: 201-web-app-github-deploy

**Pull request merged**